### ISSUE TITLE
Add to docs note about headers that have conflict with Firebase Auth

### ISSUE THREAD

**Issue by andrevferreiraa** [Baroshem/nuxt-security#127]
I just spent hours trying to debug an issue the Firebase Auth signInWithPopup.

I was getting the error `(auth/popup-closed-by-user)` and after trial and error I realised it was because of the headers `crossOriginOpenerPolicy` & `crossOriginEmbedderPolicy` set by nuxt-security. It started working after I set them to false.

Just thought it could be added to the documentation since it's likely more people will encounter this.

**Comment by Baroshem** [comment]
Hey @andrevferreiraa 

This is rather a very specific use case. I dont think that it needs to be added to the general docs about the module as the aim of these docs is to provide generic guidance about the usage of the module. 

Apart from your example there amy probably hundreds of different cases that would be beneficial to added to the docs. And then, the documentation would become really difficult to operate.

What I can recommend right now is to not add it to the docs. If the future users will have similar problems as you have, they will probably open up the module repository and search in the issues for Firebase. And then, they will see this issue with your explanation. 

I think then, this issue can be closed as not planned but will be in the history for the future users

Let me know if that works for you :)

**Comment by andrevferreiraa** [comment]
> 

Yes, my main goal with this was actually for people to see this issue when they search for this specific problem. Hopefully it will help them. You can close it up :) 

Cheers

**Comment by Baroshem** [comment]
Thanks!

Also, if you have any ideas or recommendations regarding the module, please let me know :)
